src/neo/SmartContract/ApplicationEngine.Callback.cs

```diff
@@ -0,0 +1,38 @@
+using Neo.SmartContract.Callbacks;
+using Neo.VM.Types;
+using System;
+using Array = Neo.VM.Types.Array;
+
+namespace Neo.SmartContract
+{
+    partial class ApplicationEngine
+    {
+        public static readonly InteropDescriptor System_Callback_Create = Register("System.Callback.Create", nameof(CreateCallback), 0_00000400, TriggerType.All, CallFlags.None);
+        public static readonly InteropDescriptor System_Callback_CreateFromSyscall = Register("System.Callback.CreateFromSyscall", nameof(CreateCallbackFromSyscall), 0_00000400, TriggerType.All, CallFlags.None);
+        public static readonly InteropDescriptor System_Callback_Invoke = Register("System.Callback.Invoke", nameof(InvokeCallback), 0_00000400, TriggerType.All, CallFlags.None);
+
+        internal void InvokeCallback(CallbackBase callback, Array args)
+        {
+            //TODO: We should check the count of the return values
```

Comment:
We should check the count of the return values. Wait for #1680

Reply:
@erikzhang Do you think that it will be required after https://github.com/neo-project/neo/pull/1706?